tests/unit/type/incr.tcl

```diff
@@ -182,4 +182,35 @@ start_server {tags {"incr"}} {
         r incrbyfloat foo [expr double(-1)/41]
         r get foo
     } {0}
+
+    foreach cmd {"incr" "decr" "incrby" "decrby"} {
+        set res {}
+        set expected {1 12}
+        if {[string match {*incr*} $cmd]} {
+            lappend expected 13
+        } else {
+            lappend expected 11
+        }
+
+        test {INCR[BY]/DECR[BY] operation should update encoding from raw to int} {
```

Comment:
the test name should probably include `$cmd` so that we don't have several tests with the same name

Reply:
I did try that, it was printing out `$cmd` itself.

Reply:
You need to use `""`, which allows substitution, instead of `{}`, which does not.